src/App.scss

```diff
@@ -0,0 +1,47 @@
+@import './shared.scss';
+
+.App {
+  @include flex-center;
+  min-height: 100vh;
+  flex-direction: column;
+  background: $background-gradient;
+}
+
+#language-sets-container {
+  @include flex-center;
+  flex-wrap: wrap;
+  min-height: 5.5vh;
+  margin-top: 1vh;
+}
+  .dataset-btn {
+    width: 150px;
+    background: transparent;
+    font: inherit;
+    border: $language-button-border;
+    border-radius: 50px;
+    padding: 6px;
+    margin: 4px 15px;
+    cursor: pointer;
```

Comment:
theres actually a autofocus prop for these buttons (so when active, it should render slightly different). 

can you add some styling for autofocus?

Reply:
also possibly just more differentiation between the buttons and everything else?
maybe add a header above the buttons saying "Language Cardsets" so its obvious what it is

Reply:
I see the autofocus and noticed yesterday that if you click a button it'll bring that blue focus on it, but that's it. Is that last bit how to add it to CSS to change that? Wouldn't it still lose whatever styling is applied after clicking away?

What do you mean by differentiation exactly? My thinking is that the buttons shouldn't be obtrusive or they'll distract from the card game (for example if we gave it a white background).

I personally don't think a header is necessary but if you would like that added let me know. We're running low on real estate space considering the main focus of the page should be the game.

Reply:
err, it should be autofocus so add styling under `&:autofocus: { }` and see if that works.

lets go without headers first, and just use styling to make it stand out a bit more.